projects/storefrontlib/src/cms-components/myaccount/order/cancellations-returns/cancellation-return-items/cancellation-return-items.component.spec.ts

```diff
@@ -0,0 +1,91 @@
+import { Component, Input, DebugElement } from '@angular/core';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
+import { ReactiveFormsModule } from '@angular/forms';
+import { I18nTestingModule } from '@spartacus/core';
+
+import { CancellationReturnItemsComponent } from './cancellation-return-items.component';
+
+const mockEntries = [
+  {
+    id: 1,
+    quantity: 5,
+    entryNumber: 1,
+    returnableQuantity: 4,
+  },
+];
+
+@Component({
+  template: '',
+  selector: 'cx-media',
+})
+class MockMediaComponent {
+  @Input() container;
+  @Input() format;
+}
+
+describe('OrderDetailItemsComponent', () => {
```

Comment:
The component name should be `'CancellationReturnItemsComponent'`

Reply:
yes!